src/Rules/Library/ButtonToggleAndExpandeCollapsePatterns.cs

```diff
@@ -16,13 +16,14 @@ public ButtonToggleAndExpandCollapsePatterns()
             this.Info.Description = Descriptions.ButtonToggleAndExpandCollapsePatterns;
             this.Info.HowToFix = HowToFix.ButtonToggleAndExpandCollapsePatterns;
             this.Info.Standard = A11yCriteriaId.NameRoleValue;
+            this.Info.ErrorCode = EvaluationCode.Error;
         }
 
-        public override EvaluationCode Evaluate(IA11yElement e)
+        public override bool PassesTest(IA11yElement e)
         {
-            var rule = Relationships.All(Patterns.Toggle, Patterns.ExpandCollapse);
+            var rule = ~(Relationships.All(Patterns.Toggle, Patterns.ExpandCollapse));
```

Comment:
Would it be any more efficient to leave rule unchanged, then negate the return value from Matches? That saves us creating a Condition object, and I think you've followed this pattern in previous changes